x-pack/plugins/observability/public/pages/slos/components/slo_list.tsx

```diff
@@ -71,9 +77,23 @@ export function SloList({ autoRefresh }: Props) {
           initialState={state}
         />
       </EuiFlexItem>
-      <EuiFlexItem>
-        <SloListItems sloList={results} loading={isLoading || isRefetching} error={isError} />
+      <EuiFlexItem grow={false}>
+        <ToggleSLOView sloView={sloView} setSLOView={setSLOView} setGridSize={setGridSize} />
       </EuiFlexItem>
+      {sloView === 'cardView' ? (
```

Comment:
How about we do this logic of showing list view vs card view one level up in this [file](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/pages/slos/slos.tsx#L97)?

Reply:
i think since SearchBar etc remain same , this is why i moved it there. Moving it further up will means, we will also have to move search bar and paging etc